src/internal/m365/collection/drive/delta_tree.go

```diff
@@ -264,12 +269,44 @@ func (face *folderyMcFolderFace) setTombstone(
 	return nil
 }
 
+// countLiveFilesAndSizes returns a count of the number of files in the tree
+// and the sum of all of their sizes.  Only includes filesthat are not
+// children of tombstoned containers.  If running an incremental backup, a
+// live file may be either a creation or an update.
+func (face *folderyMcFolderFace) countLiveFilesAndSizes() (int, int64) {
+	return countFilesAndSizes(face.root)
+}
+
+func countFilesAndSizes(nodey *nodeyMcNodeFace) (int, int64) {
+	if nodey == nil {
+		return 0, 0
+	}
+
+	var (
+		c int
+		s int64
```

Comment:
please use actual names here. Neither this function nor `countLiveFilesAndSizes` names the return parameters (which is fine, named return parameters are confusing for other reasons) and these names aren't descriptive. I'm left guessing which return parameter represents size and which represents the count of live files until I read some of the final lines of this function

Reply:
done; also updated the return value to a struct for clearer results.